WordPress/gutenberg
# [RNMobile] Correct ios back icon

Fixes: wordpress-mobile/gutenberg-mobile#2528

## Description

Merged [PR](https://github.com/WordPress/gutenberg/pull/23922) introduced the regression in back ios icon. That PR is fixing that



## How has this been tested?

1. Open mobile app
2. Add `Button` block
3. Open its settings
4. Go to color settings
5. Expect that back button is the same as on the screenshot presented below






## Screenshots 

<img width="492" alt="Screenshot 2020-11-04 at 12 14 22" src="https://user-images.githubusercontent.com/22746080/98105081-b0339e80-1e97-11eb-8b2f-8a8e0185980f.png">


## Types of changes





## Checklist:
